projects/xpla/index.js

```diff
@@ -0,0 +1,35 @@
+const axios = require('axios');
+
+async function staking(timestamp, ethBlock, chainBlocks) {
```

Comment:
sorry we are not tracking native chain staking for other chains, once I get some time, will fix this and make it as a metric on each chain page

Reply:
I'll delete it !